src/StoreActions.tsx

```diff
@@ -356,7 +354,6 @@ export const StoreActions = (props: {
                     diff,
                     action: { type: actionType },
                   } = item;
-                  console.log(actionType, diff && rootItem(diff, filterPath));
```

Comment:
@Aeolun not sure if you want to keep this or not

Reply:
Nah, don't want to keep any.